docs/vsphere-golden-image.md

```diff
@@ -176,12 +186,9 @@ Add the cloned Windows virtual machine as a template in the [machineset](../READ
       template:
         spec:
           providerSpec:
-            template: <Cloned Windows Virtual Machine>
+            template: <Windows Virtual Machine Template Name>
```

Comment:
We also need to call out that the network used here needs to match the network that other Linux workers are on. You need to mention this in [vsphere-golden-image.md](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/vsphere-golden-image.md) also.

Reply:
Not quite sure about such level of details, do you anticipate calling out the infrastructure ID and node labels too? Those are presented in the [documentation](https://docs.openshift.com/container-platform/4.7/machine_management/creating_machinesets/creating-machineset-vsphere.html#machineset-yaml-vsphere_creating-machineset-vsphere) as well.

Reply:
Without this "level of detail", a customer could launch a VM connected to a different network and open a bug saying it is not getting configured. So this is a missing piece in our docs.

Reply:
Maybe a better spot for this would be [configuring-windows-instances-provisioned-through-machinesets](https://github.com/openshift/windows-machine-config-operator#configuring-windows-instances-provisioned-through-machinesets).

Reply:
Agree, leaving [vsphere-golden-image.md ](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/vsphere-golden-image.md) with direct emphasis on golden image creation and usage.

Reply:
> Maybe a better spot for this would be [configuring-windows-instances-provisioned-through-machinesets](https://github.com/openshift/windows-machine-config-operator#configuring-windows-instances-provisioned-through-machinesets).

I feel that should come in a separated PR, thoughts?

Reply:
I am fine with that.

Reply:
[PR](https://github.com/openshift/windows-machine-config-operator/pull/545) opened